packages/tokens/src/types.ts

```diff
@@ -123,6 +123,8 @@ export type ColorTokens = {
   colorBrandBackground2: string;
   colorBrandBackground2Hover: string;
   colorBrandBackground2Pressed: string;
+  colorBrandBackground3Static: string;
```

Comment:
To confirm with @daisygeng - there is nothing like `colorBrandBackground2Static`.

Reply:
Discussed async, this is intentional to match `colorBrandBackground3` and `colorBrandBackground4`